mced.spec

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Since this is first public, it should probably be 2020

Reply:
Change to 2020.